Add Apple Pay Adyen App docs (#830)
saleor/saleor-docs: docs/developer/app-store/apps/adyen.mdx

:::

### Apple Pay `onValidateMerchant`

> In case you want to implement Apple Pay integration through Adyen and use your own Apple Pay certificate, you need to implement `onValidateMerchant` (Web, React Native) or `onvalidatemerchant` (iOS). Saleor App Adyen provides a way to validate the merchant through the `paymentGatewayInitialize` mutation:

Reviewer: ```suggestion
To implement Apple Pay integration through Adyen and use your own Apple Pay certificate, you must implement `onValidateMerchant` (Web, React Native) or `onvalidatemerchant` (iOS). The Adyen Saleor App provides a way to validate the merchant using the `paymentGatewayInitialize` mutation:
```
